README.md

```diff
@@ -212,26 +227,10 @@ Dashboard >> Settings >> Account
 ```yaml
   - name: GOOGLE_ANALYTICS_ID
     value: $GOOGLE_ANALYTICS_ID # Set in GitHub's Environment Secrets
-  - name: GOOGLE_OAUTH_APP_NAME
```

Comment:
I think we should restore this section (within the `yaml` code block as this reflects how we configure the deploy template).